crates/starknet/src/starknet/predeployed.rs

```diff
@@ -20,7 +28,26 @@ pub(crate) fn create_erc20() -> Result<SystemContract> {
     Ok(erc20_fee_contract)
 }
 
-pub(crate) fn create_udc20() -> Result<SystemContract> {
+/// Set initial values of ERC20 contract storage
+pub(crate) fn initialize_erc20(state: &mut StarknetState) {
+    let contract_address =
+        ContractAddress::new(Felt::from_prefixed_hex_str(ERC20_CONTRACT_ADDRESS).unwrap()).unwrap();
+    [
+        ("ERC20_name", get_selector_from_name("ether").unwrap().into()),
+        ("ERC20_symbol", get_selector_from_name("ETH").unwrap().into()),
+        ("ERC20_decimals", 18.into()),
+        // necessary to set - otherwise minting txs cannot be performed
+        ("Ownable_owner", Felt::from_prefixed_hex_str(CHARGEABLE_ACCOUNT_ADDRESS).unwrap()),
+    ]
+    .into_iter()
+    .for_each(|(storage_var_name, storage_value)| {
+        let storage_var_address = get_storage_var_address(storage_var_name, &[]).unwrap();
+        let storage_key = ContractStorageKey::new(contract_address, storage_var_address);
+        state.change_storage(storage_key, storage_value).unwrap();
```

Comment:
change_storage and get_storage_var_address could fail.

Reply:
Btw If you are going to handle the result using ? sign, in the for_each method this cannot be done. You should use a simple for loop for that